examples/sinus_fit/sinus_fit.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+    Copyright (c) 2018, Etienne de Foras and the respective contributors
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
```

Comment:
@edeforas Even though you are the one who added this file, I prefer to see uniformed file header copyright.
```
/*
    Copyright (c) 2013, Taiga Nomi and the respective contributors
    All rights reserved.
    Use of this source code is governed by a BSD-style license that can be found
    in the LICENSE file.
*/
```
My concern is, if each of the contributors used different copyright notices, that might lead to confusions.

By the way, `AUTHORS` file can be updated by script. Please see https://github.com/tiny-dnn/tiny-dnn/issues/228#issuecomment-233293328